### PR TITLE
fix weight quantization

### PR DESCRIPTION
**Pull request by elad-c**
Weight threshold calculation should always be assuming signed quantization

### CODE REVIEW

File: model_compression_toolkit/core/common/quantization/quantization_params_generation/power_of_two_selection.py [hunk 1]
```diff
@@ -54,7 +55,7 @@ def power_of_two_selection_tensor(tensor_data: np.ndarray,
         tensor_max = get_tensor_max(tensor_data, per_channel, channel_axis)
         threshold = max_power_of_two(tensor_max, min_threshold)
     else:
-        signed = np.any(tensor_data < 0)
+        signed = True  # weights are always signed
```
Comment (haihabi) [inline model_compression_toolkit/core/common/quantization/quantization_params_generation/power_of_two_selection.py:58]: Please also check symmetric quantization

**Comment by haihabi** [comment]
Add a unit tests that checks the quantizer are always signed.

**Review by reuvenperetz — APPROVED** [review]

File: model_compression_toolkit/core/common/quantization/quantization_params_generation/power_of_two_selection.py (hunk 1 above)
Comment (elad-c) [reply model_compression_toolkit/core/common/quantization/quantization_params_generation/power_of_two_selection.py:58]: isn't power of 2 always symmetric?

**Comment by reuvenperetz** [comment]
> Add a unit tests that checks the quantizer are always signed.

Maybe we should just add a TODO for now until an initial exporter version is out, so things like that can be easily checked.

**Pull request merged**